<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>742</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>mainWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>           Salidas Analógicas - PWM</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QDial" name="dailAnalog2">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Salida A2 [PWM]</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QDial" name="dailAnalog1">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_1">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Salida A1 [PWM]</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>851</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="title">
    <string>                                                  Entradas Analógicas</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>831</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <property name="spacing">
      <number>11</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Entrada A2  [0 a 10V]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Entrada A7  [0 a 10V]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLCDNumber" name="AN5"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Entrada A5  [0 a 10V]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Entrada A3  [0 a 10V]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="AN3"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Entrada A1  [0 a 10V]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLCDNumber" name="AN8"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Entrada A8  [0 a 10V]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="AN1"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="AN2"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Entrada A4  [0 a 10V]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>14</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Entrada A6  [0 a 10V]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="AN6"/>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="AN4"/>
     </item>
     <item row="3" column="2">
      <widget class="QLCDNumber" name="AN7"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Instrumento Virtual 2023</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>821</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>370</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="Iniciar">
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Finalizar">
    <property name="text">
     <string>FINALIZAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>821</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_31">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>D0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_32">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>D1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_33">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>D2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_34">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>D3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_35">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>D4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_36">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>D5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_37">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>D6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_38">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>D7</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLCDNumber" name="DispD0"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="DispD1"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="DispD2"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="DispD3"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="DispD4"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="DispD5"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="DispD6"/>
      </item>
      <item>
       <widget class="QLCDNumber" name="DispD7"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Entradas Digitales</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>821</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>D0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>D1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_17">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>D2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_18">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>D3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_19">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>D4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_20">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>D5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_21">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>D6</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_22">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>D7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="8">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="D0">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D1">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D2">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D3">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D4">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D5">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D6">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="D7">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salidas Digitales</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dailAnalog1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_13</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>927</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>960</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dailAnalog2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_14</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1118</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>1134</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
